Cargo.toml

```diff
@@ -14,12 +14,17 @@ readme = "README.md"
 
 [dependencies]
 anyhow = "1"
+async-stream = "0.3"
 chrono = "0.4"
 clap = "2"
+futures = "0.3"
 http = "0.2"
 handlebars = "3"
 hyper = { version = "0.14", features = ["full"] }
+hyper-rustls = "0.22"
```

Comment:
`hyper-rustls` is a library for HTTPS clients, not servers. I don't think you even use this dependency in the code.

Reply:
Oh! You are totally right! Thanks!